modules/email/src/services/email.service.ts

```diff
@@ -54,7 +61,12 @@ export class EmailService {
     });
   }
 
-  async sendEmail(template: string, params: ISendEmailParams) {
+  async sendEmail(
```

Comment:
this function already supports sending emails with pre-compiled body, by providing the body param in the params object. To do that you need to set template to null. Obviously that is currently not supported through typescript since template is string. Changing template type to string | undefined will allow you to provide the body and subject params without issue.